src/main/java/org/primefaces/component/dataview/DataViewBase.java

```diff
@@ -44,7 +44,8 @@
         listIcon,
         multiViewState,
         gridRowStyle,
-        gridRowStyleClass
+        gridRowStyleClass,
+        usePrimeFlex
```

Comment:
I wonder if this attribute should be less specific and make it like datatable layout="grid" or layout="flex"

Reply:
That way we can make a consistent property in all the components?

Reply:
I had the same thoughts but no good answer yet. DataView comes with list and grid - view. You can toggle between both. Layout-attribute defines which one is shown per default.
No we have two technical ways how we render grid - view. 
I am not happy with my attribute-naming. But i am pretty sure we can not add this to the existing layout-attribute.

Reply:
So PanelGrid has layout https://primefaces.github.io/primefaces/8_0/#/components/panelgrid?id=responsive

_Displays data in a 'tabular' layout, 'grid' layout or 'flex' layout. The grid and flex layout are responsive layouts. Default value is 'tabular'._

Why can't you just add the option "flex" to the layout instead of this new property?

Reply:
Yep. I was the one who added layout="flex" to PanelGrid.
PanelGrid looks almost same in all three layout-modes. The difference is its html-rendering. html-table vs. div with grid-css-classes vs div with primeflex-css-class.

DataView is IMO a different thing.
http://primefaces.org/showcase/ui/data/dataview/paginator.xhtml

It has two layouts which look fundamental different. List and Grid. 
You have a toggle switch.
![image](https://user-images.githubusercontent.com/10461942/94307237-b4090280-ff74-11ea-94c7-ab39675d50d3.png)

In XHTML you do something like
```
<p:dataView>
   <p:dataViewGridItem>
     </p:dataViewGridItem>
     <p:dataViewListItem>
     </p:dataViewListItem>
</p:dataView>
```

Now we have to technical "flavours" of grid. But from an enduser-perspective it´s still the same grid.

From a somehow isolated perspective i´d like to deliver PrimeFlex as part of PrimeFaces. (Like Grid CSS comes with PrimeFaces) And just use internal PrimeFlex because it "does the right thing". And no need to offer Grid CSS and Prime Flex - modes . And bring PrimeFlex on your own into the project, ...
But i understand the arguments why this is a bad idea.

Maybe it´s better to introduce a new Context-Param which tells PrimeFaces to use PrimeFlex whereever it can instead of Grid CSS.

Or...

Reply:
OK I see the issue now.

Reply:
We could make it like `RTLAware` and `ComponentUtils.isRTL` have a `FlexAware` and `ComponentUtils.isFlex` and that way you can set it with Context Param or with the individual component with a `flex=true` property.